gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/grpc/goldens/EchoServiceSelectiveGapicClient.golden

```diff
@@ -438,8 +438,8 @@ public class EchoServiceShouldGeneratePartialUsualClient implements BackgroundRe
    * }</pre>
    *
    * @param request The request object containing all of the parameters for the API call.
-   * @throws com.google.api.gax.rpc.ApiException if the remote call fails @InternalApi This method
-   *     is internal used only. Please do not use directly.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails  
+   * @InternalApi This method is internal used only. Please do not use directly.
```

Comment:
Sorry raising this a bit late. But I wonder if it makes more sense to add this note as part of the descriptive text, instead of as tag. Javadoc usually expects standard block tags as listed in [this guide](https://engdoc.corp.google.com/eng/doc/devguide/java/practices/javadoc.md?cl=head#at_clauses)